storage/storage-samples/storage-owner-sample/src/test/java/org/fido/iot/storage/To2StorageTest.java

```
@@ -471,6 +380,13 @@ protected void failed(Exception e) {
       if (server != null) {
         server.stop();
       }
+      try {
+        // cleanup serviceinfo files that were created during test execution
+        Files.deleteIfExists(Paths.get(System.getProperty("user.dir"), packageName));
+        Files.deleteIfExists(Paths.get(System.getProperty("user.dir"), boolName));
+      } catch (IOException e) {
+        // ignore
```

Empty catch block would result in a static code analysis issue.